tests/Sdk/Unit/Metrics/MeterTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Sdk\Unit\Metrics;
+
+use OpenTelemetry\Sdk\Metrics\Counter;
+use OpenTelemetry\Sdk\Metrics\Meter;
+use OpenTelemetry\Sdk\Metrics\UpDownCounter;
+use OpenTelemetry\Sdk\Metrics\ValueRecorder;
+use PHPUnit\Framework\TestCase;
+
+class MeterTest extends TestCase
+{
+    public function testMeterInstrumentCreation()
+    {
+        $meter = new Meter('Meter', '0.1');
+        $this->assertInstanceOf(Meter::class, $meter);
+
+        $meter_name = $meter->getName();
+        $this->assertSame('Meter', $meter_name);
+
+        $meter_version = $meter->getVersion();
+        $this->assertSame('0.1', $meter_version);
+
+        $counter = $meter->newCounter('Counter', 'A counter');
+        $this->assertInstanceOf(Counter::class, $counter);
```

Comment:
I would also suggest you test that metric's name and description are passed to the `$counter` correctly
```
$counterName = 'Counter';
$counterDescription = 'A counter';
$counter = $meter->newCounter($counterName, $counterDescription);
$this->assertInstanceOf(Counter::class, $counter);
$this->assertEquals($counterName, $counter->getName());
$this->assertEquals($counterDescription, $counter->getDescription());
```

Reply:
I would also recommend brake this one huge test into 4 different tests: one for checking Meter name and version, the other thee - one for every metric type (test a metric is created and name and description are passed correctly)

Reply:
Good call! I've split those out into their own tests.